src/server/accessors/Notifier.ts

```diff
@@ -46,4 +71,27 @@ export class Notifier implements INotifier {
     public getMessageBuilder(): IMessageBuilder {
         return new MessageBuilder();
     }
+
+    /**
+     * Creates a new direct message room between two users.
+     *
+     * @param {IUser} user - The first user to create the room for.
+     * @param {IUser} sender - The second user to create the room for.
+     * @returns {Promise<IRoom>} A Promise that resolves with the newly created direct message room.
+     */
+    private async createDMRoom(user: IUser, sender: IUser): Promise<IRoom> {
+        const newDMRId = await this.roomBridge.doCreate(
```

Comment:
you have a createDirectMessage method available in room bridge iirc

Reply:
but maybe that's hidden behind create, don't remember exactly.

Reply:
Oh, no, there isn't any `createDirectMessage` method in the room bridge, but there is a meteor method with that name, which is used inside the `create` (on the Apps.Engine calls it using the `doCreate`) room bridge.
https://github.com/RocketChat/Rocket.Chat/blob/ae5f958712cb31640fa704756382f6b7b061dc00/apps/meteor/app/apps/server/bridges/rooms.ts#L32-L33